src/SprykerEco/Glue/UnzerRestApi/Processor/Mapper/CheckoutDataResponseMapper.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Glue\UnzerRestApi\Processor\Mapper;
+
+use Generated\Shared\Transfer\RestCheckoutDataResponseAttributesTransfer;
+use Generated\Shared\Transfer\RestCheckoutDataTransfer;
+
+class CheckoutDataResponseMapper implements RestCheckoutDataResponseAttributesMapperInterface
+{
+    /**
+     * @param \SprykerEco\Glue\UnzerRestApi\Processor\Mapper\RestCheckoutDataResponseAttributesTransfer $restCheckoutDataResponseAttributesTransfer
+     * @param \SprykerEco\Glue\UnzerRestApi\Processor\Mapper\RestCheckoutDataTransfer $restCheckoutDataTransfer
+     *
+     * @return \SprykerEco\Glue\UnzerRestApi\Processor\Mapper\RestCheckoutDataResponseAttributesTransfer
+     */
+    public function mapRestCheckoutDataTransferToRestCheckoutDataResponseAttributesTransfer(
+        RestCheckoutDataResponseAttributesTransfer $restCheckoutDataResponseAttributesTransfer,
```

Comment:
method arguments order must be exchanged
map {1st param} to {2nd param}

Reply:
Fixed.